Assets/Scripts/Augment/PlayerHand.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerHand : MonoBehaviour
+{
+    [SerializeField]
+    private SkinnedMeshRenderer handMaterial;
+    [SerializeField]
+    private GameObject mesh;
+
+    public void SetPlayer(PlayerManager player)
```

Comment:
I feel like this "setter that MUST be called elsewhere after a certain point" pattern is something we should consider avoiding... Perhaps we could create a delegate to serve the same purpose?

Reply:
The purpose is to avoid issues with script execution order, but yes a delegate would definetly serve the same purpose (in a better way too). I would say this issue is very tied to #126, so I'll leave this as is for now I think. (Makes it easier for me when searching for all SetPlayer functions when refactoring later 😅)

Reply:
Yeah no need to fix it in this PR, I just came up with an improvement for this very thing that has annoyed me for a while